### DISCUSSION TITLE
Disable right-side ribbon while keeping left side's

### DISCUSSION THREAD

**Discussion by Gewerd-Strauss** [chrisgrieser/shimmering-focus#201] [Q&A] [answered]
Hello, 

is it possible via StyleSettings/could it be implemented to disable the right edge ribbon, but keep the left one? The reasoning is that the left side contains the icon's for a few plugins and has thus some merits to "being there". The right side ribbon, to my knowledge, doesn't do that. it only has the fold-arrow for hiding/showing the right sidebar, and that is really not worth keeping for me (I have both sidebars mapped, so I don't care for it and would rather get the 2-or-so% width added to my editor panes instead).

Can this be implemented? 

Thank you.
Sincerely,  
~Gw

**Comment by chrisgrieser** [comment, marked as answer]
Hi, actually there is no need to implement this, since Obsidian 0.16 (currently still in Insider) gets rid auf the right ribbon exactly for that reason ;)

So from 0.16 on, the respective Style Setting will only hide/show the left sidebar

**Reply by Gewerd-Strauss** [reply]
I don't follow development (absolutely definitely no time for that :P) so I didn't know about this, but that is great news. 

Thank you.
